backend/src/main/kotlin/com/tngtech/apicenter/backend/config/WebConfiguration.kt

```diff
@@ -1,17 +1,30 @@
 package com.tngtech.apicenter.backend.config
 
+import com.tngtech.apicenter.backend.connector.acl.service.SpecificationPermissionManager
+import com.tngtech.apicenter.backend.connector.rest.security.JwtAuthenticationProvider
+import com.tngtech.apicenter.backend.domain.exceptions.SpecificationNotFoundException
+import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
+import org.springframework.security.acls.domain.BasePermission
+import org.springframework.util.AntPathMatcher
 import org.springframework.web.servlet.config.annotation.CorsRegistry
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry
 import org.springframework.web.servlet.config.annotation.ViewControllerRegistry
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+import java.lang.Long.parseLong
 
 @Configuration
 class WebConfiguration {
 
     @Bean
     fun webMvcConfigurer(): WebMvcConfigurer {
```

Comment:
Can't we get the `AclInterceptor` as a dependency by using `fun webMvcConfigurer(aclInterceptor: AclInterceptor)` instead of field wiring?